src/main/java/uk/ac/ebi/ampt2d/registry/entities/Phenotype.java

```diff
@@ -47,28 +48,67 @@
         HEPATIC
     }
 
-    @ApiModelProperty(position = 1)
+    public enum Type {
+        DICHOTOMOUS,
+        MULTICHOTOMOUS,
+        CONTINUOUS
+    }
+
+    @ApiModelProperty(position = 1, required = true)
     @JsonProperty
     @NotNull
     @Size(min = 1, max = 255)
     @Id
     @Column(nullable = false, unique = true, updatable = false)
     private String id;
 
-    @ApiModelProperty(position = 3)
+    @ApiModelProperty(position = 2, required = true)
     @JsonProperty
     @NotNull
     @Enumerated(EnumType.STRING)
     @Column(nullable = false)
     private Group phenotypeGroup;
 
+    @ApiModelProperty(position = 3, required = true)
+    @NotNull
+    @NotBlank
+    @JsonProperty
+    @Column(nullable = false, columnDefinition = "TEXT")
+    private String description;
+
+    @ApiModelProperty(position = 4, required = true)
+    @JsonProperty
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private Type type;
+
+    @ApiModelProperty(position = 5, required = true)
+    @JsonProperty
+    @NotNull
+    @Column(nullable = false)
+    private String allowedValues;
```

Comment:
What is `allowedValues` used for?

Reply:
like if dichotomous 0 or 1, continuous is then positive or negative or float, multichotomous then what are the allowed values for the phenotype.

Reply:
Is there going to be a lot of phenotypes which are multichotomous? I wonder if we can enumerate the allowed values.

Reply:
I thought of making enumerated but we have continuous values which in the phenotype excel represent is n.n nn.nn kind of things so i left to be string.

Reply:
I would suggest renaming it as `allowedMultichtomousValues` for the sake of clarity.

Reply:
Sorry, it is a description of allowed values, like for t2d allowedValues is 0/1, DBP its nn.nn etc.